app/views/admin/users/_form.html.haml

```diff
@@ -6,23 +6,38 @@
         - user.errors.full_messages.each do |message|
           %li= message
 
-  %h4.ui.dividing.header User Info
+  .ui.equal.width.grid
+    .ui.column
+      %h4.ui.dividing.header User Info
 
-  .required.field
-    = f.label :username
-    = f.text_field :username, aria: { describedby: "Users Username" }
+      .required.field
+        = f.label :username
+        = f.text_field :username, aria: { describedby: "Users Username" }
 
-  .required.field
-    = f.label :email
-    = f.email_field :email, aria: { describedby: "Users Email" }
+      .required.field
+        = f.label :email
+        = f.email_field :email, aria: { describedby: "Users Email" }
 
-  - if user.new_record?
-    .required.field
-      = f.label :password
-      = f.password_field :password, aria: { describedby: "Users Password" }
+      - if user.new_record?
+        .required.field
+          = f.label :password
+          = f.password_field :password, aria: { describedby: "Users Password" }
 
-    .required.field
-      = f.label :password_confirmation
-      = f.password_field :password_confirmation, aria: { describedby: "Users Password Confirmation" }
+        .required.field
+          = f.label :password_confirmation
+          = f.password_field :password_confirmation, aria: { describedby: "Users Password Confirmation" }
+
+    .ui.column
+      %h4.ui.dividing.header Roles
+
+      - if current_user == user
+        %p For safety reasons, you aren't allowed to edit your own roles. Please have another Admin do so for you.
```

Comment:
hm, does this mean you need another admin :D

Reply:
🤔